README.md

```diff
@@ -45,57 +33,48 @@ When the service is running it can be checked using the ping endpoint:
       "version": "1.0.0"
     }
 
-The weather endpoint has the form:
 
-    curl http://<host:ip>/<city>/<date>/<hour minute>/
+# Pair-programming tasks
 
-    curl http://localhost:5000/london/20171005/2200/
-    {
-        "description": "broken clouds",
-        "humidity": "66%",
-        "pressure": 1027.51,
-        "temperature": "285.25"
-    }
+In the first 30 minutes you are expected to implement:
+- Implement the get forecast endpoint.
+- Build and run the API inside a docker container.
```

Comment:
Build and run the flask application inside a docker container and verify the endpoint is working as expected, using a curl request